### ISSUE TITLE
Swing die selection page should show recipe and button links

### ISSUE THREAD

**Issue by irilyth** [buttonmen-dev/buttonmen#1664]
On the swing die selection page, the button names aren't links to the button pages, and the buttons' full recipes aren't shown anywhere (e.g. if they have Reserve dice). It'd be nice if that info was there; just having a link would at least make it easy to look up in a separate tab/window.

**Comment by irilyth** [comment]
I thought of an actual use case for this: If you're setting Swing dice, and thinking ahead to what you might want both this round and after the opponent gets a Reserve die (if you win).

(Just wanted to add this in case anyone reading this was wondering "why on earth would you ever care".)

**Comment by cgolubi1** [comment]
Irilyth: is this a duplicate of the just-resolved #1245?

**Comment by irilyth** [comment]
Does #1245 include showing the button recipes? If not, I think it'd still be nice to add them, although having the links may be enough. (I guess originally I thought it was enough. :^) But in fact I think now that it'd be convenient to have that info right there, if it fits nicely.)

**Comment by dwvanstone** [comment]
No, #1245 is just about having the button names link to their pages.
